index.js

```diff
@@ -0,0 +1,7 @@
+const app = require('./src/app');
+
+const APP_PORT = process.env.PORT || 4000;
+
+app.listen(APP_PORT, () => {
+  console.log(`App is listening on port ${APP_PORT}`);
```

Comment:
Dont need to use backticks if not using a template literal, can replace these with single quotes